app/uk/gov/hmrc/tariffclassificationfrontend/controllers/SampleController.scala

```diff
@@ -54,16 +54,19 @@ class SampleController @Inject()(override val verify: RequestActions,
 
   override protected def status(c: Case): Option[SampleStatus] = c.sample.status
 
-  override protected def chooseStatusView(c: Case, notFilledForm: Form[Option[SampleStatus]])
+  protected def chooseStatusView(c: Case, notFilledForm: Form[Option[SampleStatus]], options: Option[String])
                                          (implicit request: Request[_]): Html = {
-    views.html.change_sample_status(c, notFilledForm)
+    if(options.contains("liability"))
```

Comment:
Why  do we need options when we can just do `if(c.application.isLiability)`?

Reply:
Because liabilities can also set sample status (i.e. need to show the original form).